### DISCUSSION TITLE
For Raspberry Pi Frigate users, what are you running?

### DISCUSSION THREAD

**Discussion by LaurenceGough** [blakeblackshear/frigate#5841] [General]
Hi all,

A question for all Frigate users running on a Raspberry Pi.

What are you using for recording storage? A 3.5" HDD plugged into usb? Some sort of NAS? Are you running the OS from it as well or elsewhere? Or something else? 

Any Raspberry Pi compute module 4 users here?

I'm currently using an older NAS and I'm getting a load of recording issues. 

Thanks in advance

**Comment by mdisabato** [comment]
I am using an 8GB RPi 4 with PoE connected over Ethernet to a Synology NAS. I am not having any write issues, but always have problems retrieving the event recordings. All recordings for that matter. I have 3 Lorex/Dahua cameras on the same PoE switch.  CPU utilization is pretty low, considering. There is a USB connected Coral that does the detection processing. The OS loads from the SD-RAM card. I plan to load it from the NAS when I get the system tuned. Right now, I'm working on moving to release 12.

**Reply by NickM-27** [reply]
Most likely your database is also on the NAS and that's what's causing slowdowns. I'd suggest moving the db to local storage (it's default location is /media/frigate/frigate.db

**Reply by mdisabato** [reply]
@NickM-27, 

*smacks forehead* Yes. It was. After I moved it to the SD-RAM card, the problem vanished. I sense either a surveillance-quality  SD-RAM or SSD coming in the future. :)

Thank you!

Michael.
